backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -950,13 +953,13 @@ def getMemberList(self) -> [_DAVResource]:
         if self.content:
             children = self.content.children
         else:
-            children = self.content_api.get_all(False, ContentType.Any, self.workspace)
+            children = self.content_api.get_all(False, CONTENT_TYPES.Any_SLUG, self.workspace)
```

Comment:
Hwat is the mechanism who permit `get_all(False, CONTENT_TYPES.Any_SLUG, self.workspace)` don't care about content type when you give `CONTENT_TYPES.Any_SLUG` ?

Reply:
That's correct.

Reply:
In fact that's nearly correct there is always a mask for displayable/non displayable content for all content query, but it should allow all slug but event now.

Reply:
ok